### PR TITLE
fix property typing

### PR DESCRIPTION
**Pull request by vitormattos**
Necessary fixes to change the typing properties from annotations to real property typing.

### CODE REVIEW

File: lib/Model/Attendee.php [hunk 1]
```diff
@@ -95,10 +95,10 @@ class Attendee extends Entity {
 	/** @var int */
 	protected $roomId;
 
-	/** @var string */
+	/** @var null|string */
```
Comment (nickvergessen) [inline lib/Model/Attendee.php:98]: this can not be null?

File: tests/php/Federation/FederationTest.php [hunk 2]
```diff
@@ -53,28 +54,28 @@ class FederationTest extends TestCase {
 	/** @var Notifications */
 	protected $notifications;
 
-	/** @var ICloudFederationProviderManager */
+	/** @var ICloudFederationProviderManager|MockObject */
```
Comment (nickvergessen) [inline tests/php/Federation/FederationTest.php:57]: The changes in this file is good

File: lib/Settings/Admin/AdminSettings.php [hunk 3]
```diff
@@ -58,7 +58,7 @@ class AdminSettings implements ISettings {
 	private $groupManager;
 	/** @var MatterbridgeManager */
 	private $bridgeManager;
-	/** @var IUser */
+	/** @var null|IUser */
```
Comment (nickvergessen) [inline lib/Settings/Admin/AdminSettings.php:61]: also good

File: lib/Controller/ChatController.php [hunk 4]
```diff
@@ -60,7 +60,7 @@
 
 class ChatController extends AEnvironmentAwareController {
 
-	/** @var string */
+	/** @var null|string */
```
Comment (nickvergessen) [inline lib/Controller/ChatController.php:63]: 👍🏼

File: lib/Model/Attendee.php (hunk 1 above)
Comment (nickvergessen) [reply lib/Model/Attendee.php:98]: lets keep this separate for now

Comment (vitormattos) [reply lib/Model/Attendee.php]: I identified and fixed the problem. Was a problem on tests that created Atendee entity without type.

File: tests/php/Controller/SignalingControllerTest.php [hunk 5]
```diff
@@ -392,6 +392,7 @@ public function testBackendRoomInvited() {
 
 		$attendee = Attendee::fromRow([
 			'permissions' => Attendee::PERMISSIONS_DEFAULT,
+			'actor_type' => 'user',
```
Comment (nickvergessen) [inline tests/php/Controller/SignalingControllerTest.php:395]:
```suggestion
			'actor_type' => Attendee::TYPE_...,
```

Comment (vitormattos) [reply tests/php/Controller/SignalingControllerTest.php:395]: `Attendee::ACTOR_USERS` ?

Comment (nickvergessen) [reply tests/php/Controller/SignalingControllerTest.php]: Yeah

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**